include/move_basic/sonar_ranger.h

```diff
@@ -0,0 +1,52 @@
+
+#include <tf2_geometry_msgs/tf2_geometry_msgs.h>
+#include <tf2_ros/buffer.h>
+
+
+#include <sensor_msgs/Range.h>
+
+
+// a single sonar sensor
+class SonarSensor
```

Comment:
I think it makes sense to use `Ranger` or `Ranger` instead of `Sonar` since our code doesn't care if it is sonar, IR, etc, as long as it publishes `sensor_msgs/Range`.

Reply:
Good point